README.md

```diff
@@ -97,6 +97,10 @@ token: "<token>"
 
 ## Example usage
 
+### Note
+Define `--terraform-binary-path` on the generate command which will ensure we're reusing the installed version of 
```

Comment:
Should we just check if it exists in the path and use that instead of downloading? Don't like making this something developers have to specify